zipline/assets/assets.py

```diff
@@ -242,15 +345,20 @@ def populate_cache(self):
         # Wipe caches before repopulating
         self.cache = {}
         self.sym_cache = {}
+        self.future_cache = {}
```

Comment:
On second thought, what do you think about renaming this future_chains_cache? That may be more clear, seeing as the items are chains, not individual futures?